R/README.md

```diff
@@ -63,5 +63,7 @@ You can also run the unit-tests for SparkR by running (you need to install the [
 The `./bin/spark-submit` and `./bin/sparkR` can also be used to submit jobs to YARN clusters. You will need to set YARN conf dir before doing so. For example on CDH you can run
 ```
 export YARN_CONF_DIR=/etc/hadoop/conf
+./bin/spark-submit --master yarn --deploy-mode client examples/src/main/r/dataframe.R
+OR
```

Comment:
Here, I'd prefer showing just your new example where `deploy-mode` is explicit. Might as well set a good example.